.github/workflows/release.yml

```diff
@@ -0,0 +1,87 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      candidateName:
+        description: 'Name of the candidate'
+        type: string
+        required: true
+      branchCommit:
+        description: 'Commit to check out. If it is the most recent commit then leave blank.'
+        type: string
+        required: false
+        default: ''
+      cherrypickCommits:
+        description: 'Comma separated commits to cherry pick'
+        type: string
+        required: false
+
+permissions:
+  contents: write
+
+jobs:
+  release:
+    name: Create Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get Monday date
+        id: date
+        run: echo "::set-output name=date::$(date -dmonday +'%Y-%m-%d')"
```

Comment:
Optional - I think this would be simpler if you just inline it where you use it. My rule of thumb is that if you're not reusing it, its probably not worth making it an output vs just inlining. I think there are a few other instances of single use output variables that I'd just inline.

That's mostly personal stylistic/readability related, so take it or leave it.

Reply:
Made changes accordingly